tests/waku_archive/test_retention_policy.nim

```diff
@@ -72,6 +78,13 @@ suite "Waku Archive - Retention policy":
       require (waitFor driver.deleteMessagesOlderThanTimestamp(ts=now)).isOk()  
       require (waitFor driver.performVacuum()).isOk()
 
+    # in bytes
+    let sizeLimit:int64 = 10122851
+    let excess = 69
```

Comment:
oh `sizeLimit` should be above 8 MB since that's the size even an empty DB has. so just to be safe I have used around 10 MB size. `excess` is just a random number :eyes: